### PR TITLE
change redraw to true for choropleth map animation

### LINKED ISSUE #1365: Map animation not uploading
https://plot.ly/javascript/map-animations/

The map animation isn't displaying the updated map or tooltips when "play" or a different year is selected, but it correctly updates when the map is double clicked. This bug seems to be appearing everywhere, including the codepen example.

### PR DESCRIPTION
**Pull request by michaelbabyn**
resolves #1365

### CODE REVIEW

File: _posts/plotly_js/animations/map-animations/2017-08-22-choropleth-animation.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Map Animations
-plot_url: https://codepen.io/bcd/embed/rzrJjb/?height=500&theme-id=15263&default-tab=result
+plot_url: https://codepen.io/plotly/embed/YoZLGQ/?height=500&theme-id=15263&default-tab=result
```
Comment (cldougl) [inline _posts/plotly_js/animations/map-animations/2017-08-22-choropleth-animation.html]: @michaelbabyn is this the correct codepen link? https://codepen.io/plotly/embed/YoZLGQ/?height=500&theme-id=15263&default-tab=result it doesn't seem to be animating to me?

Comment (michaelbabyn) [reply _posts/plotly_js/animations/map-animations/2017-08-22-choropleth-animation.html]: Yeah, it is. The data doesn't change very much year to year but you should notice that it works if you go from 2016 to 2010.

Comment (cldougl) [reply _posts/plotly_js/animations/map-animations/2017-08-22-choropleth-animation.html]: do you think we could use different data that does visibly change to make this example a bit stronger? (even hovering on the data it didn't really look like the #s were changing).

Comment (michaelbabyn) [reply _posts/plotly_js/animations/map-animations/2017-08-22-choropleth-animation.html]: Yes, I think that would be a much better fix.

**Comment by michaelbabyn** [comment]
I switched the dataset to gapminder here to increase the contrast in the animation and there was some other stuff wrong with this example that was preventing the hover from changing that is also fixed now.

cc @cldougl

**Pull request merged**